packages/envconfig/src/index.ts

```diff
@@ -0,0 +1,20 @@
+export {
+  loadClientConfig,
+  loadClientConfigProfile,
+  loadClientConnectConfig,
+  loadClientConfigFromToml,
+  clientConfigToToml,
+  toClientOptions,
+} from './envconfig';
+
+export {
+  ClientConfig,
+  ClientConfigProfile,
+  ClientConfigTLS,
+  LoadClientConfigOptions,
+  LoadClientProfileOptions,
+  ClientConfigFromTomlOptions,
+  ConfigDataSource,
+} from './types';
+
+export { fromTomlConfig, fromTomlProfile, toTomlConfig, toTomlProfile, loadConfigData } from './utils';
```

Comment:
Do we really want to expose all of those functions publicly? If so, we'll also need to expose the `Toml*` types.

But I'd preach to avoid conflating the public API surface at this point.

Reply:
Yes. We have followed a pattern across SDKs to allow users to convert to/from TOML representations on their own.